test/parallel/test-util-inspect.js

```diff
@@ -668,9 +669,8 @@ if (typeof Symbol !== 'undefined') {
   subject = [1, 2, 3];
   subject[Symbol('symbol')] = 42;
 
-  assert.strictEqual(util.inspect(subject), '[ 1, 2, 3 ]');
-  assert.strictEqual(util.inspect(subject, options),
-                     '[ 1, 2, 3, [length]: 3, [Symbol(symbol)]: 42 ]');
+  assert.strictEqual(util.inspect(subject),
+                     '[ 1, 2, 3, [Symbol(symbol)]: 42 ]');
```

Comment:
So in essence, this assertion is not instead of the one in 672, but instead of the one in 671. I guess it shows that symbol key properties can coexist peacefully with other properties in inspection. Or something about the order of properties? In any case, I'm guessing that the essence is preserved.